problem8.py

```diff
@@ -0,0 +1,54 @@
+"""
+Created on Sun Mar 18 19:29:43 2018
+@author: patricio
+
+find_all_groups_adj: given a string of digits and the size of the groups for
+adjacent numbers (n) it computes all posible groups to form.
+
+multiply_digits_num: multiply the digits of a numeric string
+"""
+
+def find_all_groups_adj(n, digits):
+    grouped_ints = []
+    all_groups_computed = False
+    cont = 0
+    while(cont+n<len(digits)):
+        try:
+            grouped_ints.append(digits[cont:n+cont])
+            cont += 1
+        except:
+            all_groups_computed = True
+    return grouped_ints
+        
+
+def multiply_digits_num(num_str):
+    mult_result = 1
+    for num in iter(num_str):
+        mult_result *= int(num)
+    return mult_result
+    
+digits = '73167176531330624919225119674426574742355349194934\
```

Comment:
Try to reformat long strings to comply with PEP8 and to leave a more readable code.